test/string_extensions_test.dart

```diff
@@ -826,13 +813,40 @@ void main() {
     expect("`".isCloseWrapChar(), true);
     expect("other".isOpenWrapChar(), false);
   });
-  test(
-      'wrap - wrap a string between two strings. If [before] is a wrap char and [after] is ommited, the method resolve after using [getOppositeChar]',
-      () {
+  test('wrap - wrap a string between two strings. If [before] is a wrap char and [after] is ommited, the method resolve after using [getOppositeChar]', () {
     expect("string".wrap("{"), "{string}");
     expect("string".wrap("("), "(string)");
     expect("string".wrap(null), "string");
     expect("string".wrap(""), "string");
     expect("string".wrap("AA", after: "BB"), "AAstringBB");
   });
+
+  test('removeLastEqual - Remove a string from the end of other string if equal [pattern]', () {
+    expect("string".removeLastEqual("ing"), "str");
+  });
+  test('removeFirstEqual - Remove a string from the beginning of other string if equal [pattern]', () {
+    expect("string".removeLastEqual("str"), "ing");
+  });
```

Comment:
This tests fails ! :)
```
removeFirstEqual - Remove a string from the beginning of other string if equal [pattern] [E]
  Expected: 'ing'
    Actual: 'string'
     Which: is different.
            Expected: ing
              Actual: string
                      ^
             Differ at offset 0
 ```
 
 You are calling the wrong function `removeLastEqual` instead of `removeFirstEqual`.

Reply:
Okay. Im using VScode. If u use it too, please, share your dart configuration.

Reply:
I'm using VScode aswell. I'm sharing the whole `settings.json` file :

```json
{
  "workbench.colorTheme": "One Dark Pro",
  "editor.fontFamily": "CaskaydiaCove Nerd Font",
  "workbench.iconTheme": "material-icon-theme",
  "editor.fontLigatures": true,
  "dart.openDevTools": "flutter",
  "explorer.confirmDragAndDrop": false,
  "explorer.confirmDelete": false,
  "editor.fontSize": 13.5,
  "editor.bracketPairColorization.enabled": true,
  "editor.guides.bracketPairs": "active",
  "terminal.integrated.fontFamily": "CaskaydiaCove Nerd Font Mono",
  "dart.previewLsp": true,
  "dart.flutterSdkPaths": [
    "D:\\flutter_dev",
    "D:\\flutter",
    "\\Users\\esentis\\Development\\flutter"
  ],
  "editor.suggestSelection": "first",
  "vsintellicode.modify.editor.suggestSelection": "automaticallyOverrodeDefaultValue",
  "workbench.startupEditor": "none",
  "[javascript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "html.format.unformatted": "esbenp.prettier-vscode",
  "editor.formatOnSave": true,
  "editor.codeActionsOnSave": {
    "source.organizeImports": true
  },
  "[dart]": {
    "editor.formatOnSave": true,
    "editor.formatOnType": true,
    "editor.rulers": [100],
    "editor.selectionHighlight": false,
    "editor.suggest.snippetsPreventQuickSuggestions": false,
    "editor.suggestSelection": "first",
    "editor.tabCompletion": "onlySnippets",
    "editor.wordBasedSuggestions": false,
    "editor.foldingStrategy": "indentation",
    "editor.defaultFormatter": "Dart-Code.dart-code",
    "editor.inlineSuggest.enabled": true,
    "editor.acceptSuggestionOnCommitCharacter": true,
    "editor.suggestOnTriggerCharacters": true,
    "editor.quickSuggestions": true
  },
  "ruby.useBundler": true, //run non-lint commands with bundle exec
  "ruby.useLanguageServer": true, // use the internal language server (see below)
  "ruby.lint": {
    "rubocop": {
      "useBundler": true // enable rubocop via bundler
    },
    "reek": {
      "useBundler": true // enable reek via bundler
    }
  },
  "ruby.format": "rubocop",
  "javascript.updateImportsOnFileMove.enabled": "always",
  "dart.showInspectorNotificationsForWidgetErrors": false,
  "terminal.explorerKind": "external",
  "git.autofetch": true,
  "html.validate.styles": false,
  "css.validate": false // use rubocop for formatting,
}
```